### PR TITLE
otonasalone.jp

### LINKED ISSUE #120795: otonasalone.jp
### Issue URL (Ads)
 [https://otonasalone.jp/291367/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fotonasalone.jp%2F291367%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.4.0
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #120795

### CODE REVIEW

**Review by Yuki2718 — COMMENTED** [review]
Probably we should change `~pinterest.com##a[href^="http://pinterest.com/pin/create/button"]` in Social general_elemhide to `~pinterest.com##a[href*="pinterest.com/pin/create/button"]`
(Possibly `~pinterest.com##a[href^="http://pinterest.com/pin/create/link/"]` might be merged?) @zloyden Pls also check for the social rule.
@SKEIDs You don't need to change until official member comes and comments.

**Comment by zloyden** [comment]
@Yuki2718 @SKEIDs  I have changed the rule.

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -74,6 +74,7 @@ beautynation.jp##.m-article-share > div.contents
 beautynation.jp##.m-article-share > div.title
 crea.bunshun.jp##.module-sns-share
 bepal.net##.shareFooter
+otonasalone.jp##.l-newscategory > div[style="float:right;"]
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:77]: The rule is ok, but not needed.

**Review by zloyden — APPROVED** [review]

**Comment by SKEIDs** [comment]
> @Yuki2718 @SKEIDs  I have changed the rule.

thank you so much

**Pull request merged**